fix generators for big argument list >50

sulphur opened the pull request:
Hello, 
I've stumbled on an issue where generated test cases weren't valid because  `IO.inspect` and `Kernel.inspect` by default inspect only 50 elements in maps. Therefore
generating a schema for example :

`mix phx.gen.live NewPerformers Performer performers attr1 ... attr55`

will result in invalid maps in test cases where `@create_attrs` etc. will be truncated by `inspect`
with `...}` at the end
like this:

```%{attr1: "some value", ..., attr50: "some value, ...}```

It is my first look in the Phoenix code base so my fix is rather naive. I think that I might be fixed in a better way like maybe writing a helper or something. But for now, I have no better idea ;) 

Cheers
Piotr

josevalim commented:
This looks great @sulphur! I wonder if we should encapsulate this in its own function and simply call it from the template. This way, if we need to change other parameters in the future, we have to change only one place. WDYT? Can you also please rebase the PR? Thanks!

sulphur commented:
Hi, I've extracted to a helper function that i've put into `Mix.Phoenix` module (not sure whether it is a good place) and i've rebased to latest `master` branch.

josevalim commented:
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:
